src/metabase/driver/druid.clj

```diff
@@ -70,17 +70,31 @@
 
 ;;; ### Sync
 
-(defn- describe-table-field [druid-field-type field-name]
+(defn- do-segment-metadata-query [details datasource]
+  (do-query details {"queryType"     "segmentMetadata"
+                     "dataSource"    datasource
+                     "intervals"     ["1999-01-01/2114-01-01"]
+                     "analysisTypes" []
+                     "merge"         true}))
+
+(defn- druid-type->base-type [field-type]
+  (case field-type
+    "STRING"      :type/Text
+    "FLOAT"       :type/Float
+    "LONG"        :type/Float
```

Comment:
Shouldn't this be `:type/Long`?

Reply:
That's a good question. @psalaberria002 ?

Reply:
Hmm. Long time since I checked. I am not sure why I did that.
I guess ir should be `:type/Long`.

Reply:
`:type/Integer` probably makes the most sense. We don't currently differentiate between 64-bit and 32-bit integers for query-writing purposes which is why there's no separate `:type/Long` yet. `:type/BigInteger` however we do use in places where things specifically map to a java `BigInteger` type. I'll change this